gateway-js/src/executeQueryPlan.ts

```diff
@@ -239,10 +239,12 @@ export async function executeQueryPlan(
           // Note that this behavior is also what the router does (and in fact, the exact name of the `extensions` we use,
           // "valueCompletion", comes from the router and we use it for alignment.
           if (errors.length === 0 && postProcessingErrors.length > 0) {
+            span.recordException(postProcessingErrors[0]);
```

Comment:
It's not obvious to me in their docs whether calling `recordException` multiple times works, but I do think it would be nice to capture all of them rather than just the first if that's an option.

Reply:
It is possible to call `recordException` multiple times to report multiple exception events. I will change it to do that then. 👍